R/g.part5.R

```diff
@@ -163,7 +163,7 @@ g.part5 = function(datadir = c(), metadatadir = c(), f0=c(), f1=c(),
       di = 1
       fi = 1
       SPTE_end = c() # if it is not loaded from part3 milestone data then this will be the default
-      if (length(idindex) > 0 & nrow(summarysleep) > 1) { #only attempt to load file if it was processed for sleep
+      if (length(idindex) > 0 & nrow(summarysleep) > 0) { #only attempt to load file if it was processed for sleep
```

Comment:
Good idea, but adding the date is not needed because the commit itself already has a timestamp and it reduces readability to dates in the code. I am now updating the comment.